api/k8s/v1/k8s.proto

```diff
@@ -553,6 +553,11 @@ message ResizeHPARequest {
     uint32 max = 2;
   }
   Sizing sizing = 5 [ (validate.rules).message = {required : true} ];
+
+  Sizing old_sizing = 6;
+
+  // A flag to add sirens to the audit message or not
+  bool add_warning = 7;
```

Comment:
and can an example be added (ie if the new sizing is much greater/less than old sizing)
```suggestion
  bool resize_warning = 7;
```